posthog/temporal/data_imports/sources/postgres/postgres.py

```diff
@@ -30,6 +31,10 @@
 from posthog.warehouse.types import IncrementalFieldType, PartitionSettings
 
 
+def is_supabase_read_replica(host: str) -> bool:
+    return host.endswith(".supabase.co") and "-rr-" in host
```

Comment:
These replication errors aren't limited to supabase: https://www.google.com/search?q=%22terminating+connection+due+to+conflict+with+recovery%22